src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -97,9 +103,9 @@ boolean verifyUsername(String toVerify) {
   }
 
   /**
-   * Prompts the user for a valid host name. Host names must be alpha numeric + ".". They cannot be
-   * longer than 255 characters. Each host name segment cannot exceed 63 characters. They must end
-   * and start with alpha numeric characters.
+   * Prompt the user for a valid host name which follows the following parameters: (1) Must be alpha
+   * numeric + "." (2) Can't be longer than 255 characters (3) Host name segments cannot exceed 63
+   * characters (4) Must end and start with alpha numeric characters
```

Comment:
Alphanumeric is one word; would you mind fixing this here? (It's shown as two words in a couple different places.)

Reply:
Will change it! Thank you